src/App.vue

```diff
@@ -13,5 +13,6 @@
 
 body {
   margin: 0;
+  min-width: 20rem;
```

Comment:
Just curious, what's the intention here?

Reply:
it was agreed with @zvenigorodskaia to restrict the body as it's the minimal size of modern mobile devices we know

Reply:
And what's the point? To make it look all right if you shrink it below 20rem or something?